CHANGELOG.next.toml

```diff
@@ -9,4 +9,10 @@
 # message = "Fix typos in module documentation for generated crates"
 # references = ["smithy-rs#920"]
 # meta = { "breaking" = false, "tada" = false, "bug" = false }
-# author = "rcoh"
\ No newline at end of file
+# author = "rcoh"
+
+[[smithy-rs]]
+message = "Use a newtyped IndexSet to back the \"set\" shape"
+references = ["smithy-rs#1446"]
+meta = { "breaking" = true, "tada" = false, "bug" = false }
+author = "hlbarber"
```

Comment:
we'll need to call out the exact breaking change and provide upgrade instructions